spec/mocks/items.ts

```diff
@@ -102,3 +102,16 @@ export const itemFragmentMock = {
   metadata: {},
   contentHash: '',
 }
+
+export function convertItemDatesToISO<T extends ItemAttributes | FullItem>(
```

Comment:
This is called convertItemDatesToISO but it takes and uses a collection. Is that an naming error?
an item does not have a `reviewed_at`